components/winners.py

```diff
@@ -21,7 +21,7 @@ def compute_winners():
 
     hidden_features = os.environ.get("HIDDEN_FEATURES")
     public = {"public": True} if not hidden_features else {}
-    df = get_tourneys(TourneyResult.objects.filter(league=champ, **public), offset=0, limit=10)
+    df = get_tourneys(TourneyResult.objects.filter(league=legend, **public), offset=0, limit=10)
```

Comment:
definitely lock as others, perhaps lock forever actually as a testament of the Champ times